linera-service/src/server.rs

```diff
@@ -364,17 +379,27 @@ async fn main() {
             max_concurrent_queries,
             max_stream_queries,
             cache_size,
+            kube,
         } => {
             let genesis_config = GenesisConfig::read(&genesis_config_path)
                 .expect("Fail to read initial chain config");
-            let server_config = ValidatorServerConfig::read(&server_config_path)
+            let mut tmp_server_config = ValidatorServerConfig::read(&server_config_path)
```

Comment:
@andresilva91 What are you trying to achieve with the `tmp` variable?

Reply:
Huh, you're right. For some reason I remember some mutability issue (like the code below expected an immutable variable), but it compiles without the tmp variable. So I'll remove it. Thanks for pointing it out!